docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -43,90 +46,76 @@ $ cd tpcc
 The binaries are compiled with JAVA 13 and it is recommended to run these binaries with that version.
 {{< /note >}}
 
-## Step 2. Start your database
-
-Start the database using the steps mentioned [here](https://docs.yugabyte.com/latest/quick-start/create-local-cluster/macos/).
-
-## Step 3. Configure connection properties
-
-Set the following connection configurations in the workload configuration file (`config/workload_all.xml`).
-
-```sh
-<!-- Connection details -->
-<dbtype>postgres</dbtype>
-<driver>org.postgresql.Driver</driver>
-<DBUrl>jdbc:postgresql://<ip>:5433/yugabyte</DBUrl>
-<username>yugabyte</username>
-<password></password>
-<isolation>TRANSACTION_REPEATABLE_READ</isolation>
-```
-
-The details of the workloads have already been populated in the sample configuration files located in the `/config` directory.
-The workload descriptor works the same way as it does in the upstream branch and details can be found in the [online documentation](https://github.com/oltpbenchmark/oltpbench/wiki).
+Start your YugabyteDB cluster by following the steps [here](../../deploy/manual-deployment/).
 
-## Step 4. Running the TPC-C benchmark
+{{< tip title="Tip" >}}
+You will need the IP addresses of the nodes in the cluster for the next step.
+{{< /tip>}}
 
-Use the provided utility script (`./tpccbenchmark`) to run the TPC-C benchmark. Available options are:
 
-```
--c,--config &lt;arg&gt;            [required] Workload configuration file
-   --clear &lt;arg&gt;             Clear all records in the database for this benchmark
-   --create &lt;arg&gt;            Initialize the database for this benchmark
-   --dialects-export &lt;arg&gt;   Export benchmark SQL to a dialects file
-   --execute &lt;arg&gt;           Execute the benchmark workload
--h,--help                          Print this help
-   --histograms                    Print txn histograms
-   --load &lt;arg&gt;              Load data using the benchmark's data loader
--o,--output &lt;arg&gt;            Output file (default System.out)
-   --runscript &lt;arg&gt;         Run an SQL script
--s,--sample &lt;arg&gt;            Sampling window
--v,--verbose                       Display Messages
-```
+### Step 2. TPC-C Load Phase
 
 Before starting the workload, you will need to load the data first.
 
+
 ```sh
-$ ./tpccbenchmark -c config/workload_all.xml --create=true --load=true
+$ ./tpccbenchmark --create=true --load=true \
+  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
+  --warehouses=100 \
+  --loaderthreads=48
 ```
 
-Then, you can run the workload.
+The default values for `nodes` is `127.0.0.1`, `warehouses` is `10` and
```

Comment:
Note box for this would be great.

Reply:
Also would be good to write a line above this box which says:

```
Remember to replace the IP addresses above with that of the nodes in the cluster.
```

Reply:
Done.